internal/bminventory/inventory.go

```diff
@@ -1446,13 +1448,11 @@ func (b *bareMetalInventory) createAndUploadNewImage(ctx context.Context, log lo
 
 	// return without generating the image if we're using the image service or if the image has already been generated
 	if b.ImageServiceBaseURL != "" {
-		if err = b.updateExternalImageInfo(infraEnv, infraEnvProxyHash, imageType); err != nil {
+		logrus.Info("XXXX: before updateExternalImageInfo")
```

Comment:
@nmagnezi think you left a debug line in here.

Reply:
yikes. removing.

Reply:
done